Add 11.10 release notes (#236)
mondoohq/docs: releases/2024-06-25-mondoo-11.10-is-out.md

---
slug: mondoo-11.10-is-out/
title: Mondoo 11.10 is out!

> description: Announcing the 11.10 release of Mondoo, with CIS benchmarks for Exchange Server 2019, faster Azure scans, new resouces, and more!

Reviewer: ```suggestion
description: Announcing the 11.10 release of Mondoo, with CIS benchmarks for Exchange Server 2019, faster Azure scans, new resources, and more!
```

> description: Announcing the 11.10 release of Mondoo, with CIS benchmarks for Exchange Server 2019, faster Azure scans, new resources, and more!

Reviewer: ```suggestion
description: Announcing the 11.10 release of Mondoo, with Windows filesystem scanning, expanded IaC reporting, and more!
```

Reviewer: Okay to make this accurate, how about:
```suggestion
description: Announcing the 11.10 release of Mondoo, with CIS benchmarks for Exchange Server 2019, improvements to the CVE and advisory UI, new resources, and more!
```

author: Tim Smith
author_title: Mondoo Core Team
author_url: https://github.com/tas50
image: /img/featured_img/release-feature.jpg
tags: [release, mondoo]
---

> ## 🥳 Mondoo 11.10 is out! This release includes CIS benchmarks for Exchange Server 2019, faster Azure scans, new resources, and more!

Reviewer: Let's accurately summarize this now that we know what is in it. :) 

```suggestion
## 🥳 Mondoo 11.10 is out! This release includes CIS benchmarks for Exchange Server 2019, improvements to the CVE and advisory UI, new resources, and more!
```

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

### Secure Microsoft Exchange Server 2019

Validate your Microsoft Exchange Server 2019 instance against the latest CIS benchmarks with our new policy, which contains 52 new tests to ensure that you follow best practices and security recommendations.

![Exchange Server 2019 Checks](/img/releases/2024-06-25-mondoo-11.10-is-out/exchange.png)

## 🧹 IMPROVEMENTS

> ### Improved CVE and advisories affected assets lists

Reviewer: ```suggestion
### Improved affected assets lists for CVE and advisories
```

- New `iamRole` field.

#### aws.vpc.routetable

- New `associations` field using the new `aws.vpc.routetable.associations` resource.

#### gcp.project.computeService

- New `enabled` field.

#### googleworkspace

- New `calendars` field using the new `googleworkspace.calendar` resource.

#### googleworkspace.report.apps

- New `admin` field.

#### googleworkspace.user

- New `isDelegatedAdmin` field.

## 🐛 BUG FIXES AND UPDATES

- Add the generation time in addition to the date on all compliance reports.
- Automatically enable the generated compliance evidence policy for custom compliance frameworks.
- Render check and data query markdown in compliance reports.
- Add the number of queries to the compliance report summary page.
- Fetch the latest VMware advisories that are now published by Broadcom.
- Improve the output of queries in Kubernetes policies.
- Improve reliability of the "Ensure 'Debug programs' is set to 'Administrators'" check in Windows policies.
- Improve connection error output for Google Cloud, Azure, and AWS resources.
- Improve filters on the Azure query pack to prevent failures running queries.
- Fix missing check counts in the asset check overview.

> - Improve reliability of the CIS Amazon EKS benchmark "Ensure Network Policy is Enabled and set as appropriate" and "Ensure clusters are created with Private Nodes" checks.

Reviewer: Well this capitalization is wonky, but I suspect that's how the check names are capitalized, so meh.

- Expand the NSA PowerShell policy's "Disable and uninstall the deprecated PowerShell v2" check to work on both desktop and server Windows installations.

> - Improve reliability of AWS IAM credential report fetching.

Reviewer: ```suggestion
- More reliably fetch AWS IAM credential reports.
```

> - Improve reliability of Azure managed identity credentials fetching.

Reviewer: ```suggestion
- More reliably fetch Azure managed identity credentials.
```

---
slug: mondoo-11.10-is-out/
title: Mondoo 11.10 is out!
description: Announcing the 11.10 release of Mondoo, with FOO, BAR, BAZ, and more!
author: Tim Smith
author_title: Mondoo Core Team
author_url: https://github.com/tas50
image: /img/featured_img/release-feature.jpg
tags: [release, mondoo]
---

## 🥳 Mondoo 11.10 is out! This release includes Windows filesystem scanning, expanded IaC reporting, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

### Secure Microsoft Exchange Server 2019

https://github.com/mondoohq/cnspec-enterprise-policies/pull/917

### Continuous IaC discovery in GitHub and GitLab

https://github.com/mondoohq/console/pull/4550

> ### Continuos GitHub Enterprise scanning

Reviewer: "Continuos" is a baroque musical term. Is our spellcheck seriously allowing it? 😆
